tests/api/Jalasoft.TeamUp.Resumes.API.Tests/GetResumesByNameTests.cs

```diff
@@ -0,0 +1,135 @@
+            this.mockHttpContext = new DefaultHttpContext();
+            this.getResumesByName = new GetResumesByName(this.mockService.Object);
+        }
+
+        public static IEnumerable<Resume> GetTestResumes()
+        {
+            var resumes = new List<Resume>
+            {
+                new Resume()
+                {
+                    Id = 1,
+                    Title = "Guido Castro",
+                    Contact = new Contact
+                    {
+                        Address = "15 street",
+                        Email = "guido.castro@fundacion-jala.org",
+                        Phone = 123456789,
+                    },
+                    CreationDate = DateTime.Now,
+                    LastUpdate = DateTime.Now,
+                    Person = new Person
+                    {
+                        FirstName = "Guido",
+                        LastName = "Castro",
+                        Birthdate = DateTime.Now,
+                        Picture = "picture.jpg"
+                    },
+                    Summary = "I'm a great developer :-D",
+                    Skills = new List<Skill>()
+                    {
+                        new Skill
+                        {
+                            Id = 1,
+                            Name = "C#"
+                        },
+                        new Skill
+                        {
+                            Id = 2,
+                            Name = "Javascript"
+                        },
+                    }
+                },
+                new Resume()
+                {
+                    Id = 2,
+                    Title = "Marcelo Ruiz",
+                    Contact = new Contact
+                    {
+                        Address = "16 street",
+                        Email = "marcelo.ruiz@fundacion-jala.org",
+                        Phone = 123456789,
+                    },
+                    CreationDate = DateTime.Now,
+                    LastUpdate = DateTime.Now,
+                    Person = new Person
+                    {
+                        FirstName = "Marcelo",
+                        LastName = "Ruiz",
+                        Birthdate = DateTime.Now,
+                        Picture = "picture.jpg"
+                    },
+                    Summary = "I'm a great ux :-D",
+                    Skills = new List<Skill>()
+                    {
+                        new Skill
+                        {
+                            Id = 1,
+                            Name = "Figma"
+                        },
+                        new Skill
+                        {
+                            Id = 2,
+                            Name = "HTML"
+                        },
+                    }
+                }
+            };
+            return resumes;
+        }
+
+        [Fact]
+        public void GetResumeByName_ValidId_Resume()
```

Comment:
GetResumeByName_ValidId_Resume
This shouldn't be "validID" since we are not receiving an ID we are receiving a name

Reply:
It's right, corrected.